services/api/src/utils/logging.js

```diff
@@ -64,13 +99,18 @@ exports.createLogger = createLogger;
 const formatters = {
   // https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry
   gcloud: function ({ request, response, latency }) {
+    const { method, url } = request;
+    const { statusCode: status } = response;
+    const length = response.getHeader('content-length');
+    const meta = `${latency}ms ${length ? bytes(+length) : '?'}`;
+    const message = `${chalk.white(method)} ${chalk.gray(url)} ${chalk.green(status)} ${chalk.gray(meta)}`;
```

Comment:
sceptical ... do we really need all these colors? They wont show up in google cloud. 

The @bedrockio/instrumentation is using console[level] for outputs (in development) isn't that enough, e.g. the console.error will display red if recall ? 

Anyway if this is really wanted for local development we can surely port that over.

Reply:
wanted to get it aligned with output of `koa-logger` and go one step further... it really helps to be able to quickly scan logs and I feel like we lost that when we unhooked it.

Good point about production though what if we just did something like this:

```
function color(name, msg) {
  return ENV_NAME === 'production' ? msg : chalk[name](msg);
}

color('red', 'error');
```

Reply:
we for sure can do that, but yeah still questioning if it worth the overhead ?

Reply:
yeah this is obtuse, I'd really like to simplify ... what I really want is actually:

```
function getLogger() {
   return ENV_NAME === 'development' ? getDevLogger() : getOpenTelemetryLoger();
}
```

keep all the functionality separate